gap/projective.gi

```diff
@@ -14,7 +14,7 @@
 
 SLPforElementFuncsProjective.TrivialProjectiveGroup :=
    function(ri,g)
-     return StraightLineProgramNC( [ [1,0] ], 1 );
+     return StraightLineProgram( [ [1,0] ], 1 );
```

Comment:
Why? I mean, OK, it probably doesn't matter either way -- but here we really know the SLP is correct, no needs for checks (that said, I have no idea what checks the NC version omits)

Reply:
Sorry, cut and paste error.